kube-controllers/pkg/converter/networkpolicy_converter.go

```diff
@@ -52,7 +54,9 @@ func (p *policyConverter) Convert(k8sObj interface{}) (interface{}, error) {
 
 	c := conversion.NewConverter()
 	kvp, err := c.K8sNetworkPolicyToCalico(np)
-	if err != nil {
+	// Silently ignore rule conversion errors
```

Comment:
I think it's worth expanding on this comment - probably should say "we don't expect any conversion errors, since the data given to us here is validated by the Kubernetes API. The conversion code ignores any rules that it cannot parse, and we will pass the valid ones to Felix."

Reply:
will add this, thanks @caseydavenport !